test/groovy/CloudFoundryDeployTest.groovy

```diff
@@ -260,12 +246,10 @@ class CloudFoundryDeployTest extends BasePiperTest {
             deployTool: 'mtaDeployPlugin',
             mtaPath: 'target/test.mtar'
         ])
-
-        assertEquals('s4sdk/docker-cf-cli', jedr.dockerParams.dockerImage)
-        assertEquals('/home/piper', jedr.dockerParams.dockerWorkspace)
-
-        assertTrue(jscr.shell[0].contains('cf login -u test_cf -p \'********\' -a https://api.cf.eu10.hana.ondemand.com -o "testOrg" -s "testSpace"'))
-        assertTrue(jscr.shell[0].contains("cf deploy target/test.mtar -f".toString()))
+        // asserts
```

Comment:
unrelated to scope of PR, only another way of asserting --> should be done in a dedicate PR.

Reply:
needed to change assertion because of `assertTrue(jscr.shell[0].contains...`